kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go

```diff
@@ -53,6 +53,9 @@ func KubeadmUpgrade(c *status.Cluster, upgradeVersion *K8sVersion.Version, patch
 		}
 
 		if n.Name() == c.BootstrapControlPlane().Name() {
+			if err := kubeadmUpgradePlan(c, n, upgradeVersion, featureGate, vLevel); err != nil {
```

Comment:
we need to state it clearly that `kubeadm upgrade plan` can only run on the control node, maybe the `Short` msg in the cobra command, not related to this pr though.

Reply:
it can be run on any node that has an admin.conf, i think.
but if you'd like to PR such a change on the cobra for "plan" please go ahead.

Reply:
handled ion separate PR
https://github.com/kubernetes/kubernetes/pull/122213